staging/src/k8s.io/apiserver/plugin/pkg/audit/webhook/webhook.go

```diff
@@ -44,9 +44,18 @@ func init() {
 	install.Install(audit.Scheme)
 }
 
+func retryOnError(err error) bool {
```

Comment:
Comment here on when it might make sense to retry on all errors (versus DefaultShouldRetry) ?

Reply:
done